.gitignore

```diff
@@ -27,4 +27,7 @@ npm-debug.log
 /config/*.secret.exs
 
 #.elixir_ls
-.elixir_ls/
\ No newline at end of file
+.elixir_ls/
+
+#.env
+.env
```

Comment:
Not sure if this is standard in Elixir/Phoenix for setting env variables locally -- a reference would help my understanding on the subject, but not required. 

The comment could also read either more informatively or not exist (I'm eyeing that .elixir_ls comment too, though I recognize that's from a previous PR)